Lesson006/Task 3 HW06.ArrayModifier/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2_HW06.ThirdArrayCreator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] arrayRandomInput = new int[5];
+            Random random = new Random();
+
+            for (int i = 0; i < arrayRandomInput.Length; i++)
+            {
+                arrayRandomInput[i] = random.Next(20);
+                Console.WriteLine("Array element value " +  " = " + arrayRandomInput[i]);
+            }
+
+            int[] arrayKeyboardInput = new int[5];
+
+            Console.WriteLine("Keyboard input: ");
```

Comment:
This task is the same as the previous task. I don't see solve task ArrayModifier.

Reply:
Changed Task3